### ISSUE TITLE
 Cannot train a model with the same accuracy as the paper's

### ISSUE THREAD

**Issue by sewichou** [f90/Wave-U-Net-Pytorch#7]
Thanks for such excellent work! The code is great and well-organized. But I met this problem during training.

I tried evaluating the SDR for both pretrained model and the model I trained with default setting, and the best overall SDR was around 2. That was 1. away from the accuracy claimed in the paper.

Is there any important setting I miss? Should I use the original tensorflow implement code to try it again?

**Comment by YadongChen-1016** [comment]
I got the same situation as you, the best overall SDR value is about 1.8.
